src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,44 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+
+    if (typeof text !== "string" || !text) {
+        throw "Please enter a valid string";
```

Comment:
Prefer to throw `Error` as in 

```js
throw Error('...')
```

to preserve the stack trace. In this case, throw a `TypeError` instead.